.circleci/config.yml

```diff
@@ -284,7 +210,9 @@ workflows:
             - plugin-wpe-headless-bundle-json
           filters:
             branches:
-              ignore: /.*/
```

Comment:
@markkelnar I made this change to get it in sync with the staging job above. Does this look good to you?

Reply:
This job will only run when the matching tag is present in the pipeline. I don't know what the branch will be and previously, I've always used the ignore * filter for branches, per circleci documentation.